packages/cosmic-proto/proto/icq/v1/icq.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package icq.v1;
```

Comment:
It's unclear to me whether these (`icq/v1`) should live here (top level) or inside `/ibc/applications`. When placed in that directory, i see `Error: Dependency Not Found icq/v1/icq.proto` during `yarn codegen`. A concern is that the correct typeUrls may not be generated with this folder structure. Its seems we'd need to update this line to package `ibc.applications.icq.v1`, but not sure there is any sort of precedent for editing these files.

cc @michaelfig for insights

Reply:
The naming for `.proto` files is relative to the most immediate `proto/` directory.  It is important not to change the `package ...` lines to match a different directory structure.  That will break the generated proto codecs.  Instead just move the files into `proto/${packageName.replaceAll('.', '/')`, as was correctly done in this PR.